generators/server/templates/src/main/java/package/service/dto/_UserDTO.java

```diff
@@ -109,6 +109,10 @@ public String getLogin() {
         return login;
     }
 
+    public void setLogin(String login) {
```

Comment:
This means the object isn't immutable anymore, why do you want to be able to change the user login? I don't think this is currently needed (the method is not called)